Kernel/cpu-6809/rules.mk

```diff
@@ -1,6 +1,6 @@
 export CROSS_AS=m6809-unknown-as
-export CROSS_LD=lwlink
-export CROSS_CC = m6809-unknown-gcc
+export CROSS_LD=m6809-unknown-ld
+export CROSS_CC=m6809-unknown-gcc
 #export CROSS_CCOPTS=-Wall -O2 -I$(ROOT_DIR)/cpu-6809 -I$(ROOT_DIR)/platform-$(TARGET) -I$(ROOT_DIR)/include
```

Comment:
The lack of whitespace around "=" is consistent with the surrounding code, but I would have liked to see a follow-up commit fixing the whitespace in this file..